spiceaidocs/docs/deployment/kubernetes/index.md

```diff
@@ -98,7 +98,35 @@ helm upgrade --install spiceai spiceai/spiceai --set monitoring.podMonitoring.en
 
 Once the monitoring is enabled, import the [Spice Grafana dashboard](../../clients/grafana/index.md) to visualize the Spice.ai metrics.
 
-### Example values.yaml
+### Health and Readyness
```

Comment:
```suggestion
### Health and Readiness
```

Also worth providing a link to K8s docs: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes